### PR TITLE
editorial: update links

### PR DESCRIPTION
**Pull request by marcoscaceres**
Fixup linking - remove some badness (e.g., reference to WebIDL-1 and HTML5).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/resource-timing/pull/170.html" title="Last updated on Oct 11, 2018, 7:11 AM GMT (b2a5f02)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/resource-timing/170/de1f8b5...b2a5f02.html" title="Last updated on Oct 11, 2018, 7:11 AM GMT (b2a5f02)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
The HTML links might be broken now... but please don't point to the outdated "HTML5".

**Comment by marcoscaceres** [comment]
Note: when you run tidy, you should run it like this: 
`tidy -config tidyconfig.txt -o index.html index.html`

File: index.html [hunk 1]
```diff
@@ -237,36 +223,36 @@ <h3>Introduction</h3>
...
 Timeline</a> of the relevant context. Resources that are retrieved
-from <a data-cite="!HTML/#relevant-application-cache">relevant
+from <a data-cite="HTML#relevant-application-cache">relevant
 application caches</a> or local resources MUST be included as
 <a>PerformanceResourceTiming</a> objects in the <a data-cite=
-"!PERFORMANCE-TIMELINE-2/#performance-timeline">Performance
-Timeline</a> [[!PERFORMANCE-TIMELINE-2]]. Resources for which the
-<a data-cite="!FETCH/#concept-fetch">fetch</a> was initiated, but
-was later aborted (e.g. due to a network error) MAY be included as
+"PERFORMANCE-TIMELINE-2#performance-timeline">Performance
```
Comment (yoavweiss) [inline index.html]: While we're at it, is it possible to point to the WD, rather than the TR? (Potentially at a different PR)

Comment (yoavweiss) [reply index.html]: To be clear, I can do that, just asking if it's possible (and if so, what's the magic spec name)

Comment (marcoscaceres) [reply index.html]: Yes, but you need to check the key on https://www.specref.org and add it if missing.

File: index.html [hunk 2]
```diff
@@ -813,13 +788,13 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
...
 <p>The <dfn>Performance</dfn> interface is defined in
-[[!HR-TIME-2]].</p>
+[[HR-TIME-2]].</p>
 <p>The method <dfn>clearResourceTimings</dfn> runs the following
 steps:</p>
 <ol>
 <li>remove all <a>PerformanceResourceTiming</a> objects in the
 <a data-cite=
-'!PERFORMANCE-TIMELINE-2/#dfn-performance-entry-buffer'>performance
+'!PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer'>performance
```
Comment (yoavweiss) [inline index.html]: So here we need the "!" prefix? If so why? WHY???????

Comment (yoavweiss) [reply index.html]: Also, can you point me to relevant docs that'd allow me to educate myself on that front?

File: index.html (hunk 1 above)
Comment (yoavweiss) [reply index.html]: Looking at specref, it seems like the ED URL is already there in the w3c.json, but as "edDraft" rather then the "href". Is it possible to point to that URL from the caller? Or do I need to add a separate entry where the "href" is the ED draft URL?

File: index.html (hunk 2 above)
Comment (marcoscaceres) [reply index.html]: Oh, you missed the memo 📝 https://lists.w3.org/Archives/Public/spec-prod/2018OctDec/0000.html

We now use “artificial intelligence” (i.e., “if statements”) for citations, so ReSpec knows if a reference is normative or informative:)

Comment (marcoscaceres) [reply index.html]: So, we don’t actually need “!” anywhere, my mistake when I did find/replace.

Comment (yoavweiss) [reply index.html]: I'm glad our robot overlords are being kind with us. Can you remove the '!' then?

Comment (marcoscaceres) [reply index.html]: Yeah, remove it. Sorry, already left the office, so on my phone... otherwise would do it now.

Comment (yoavweiss) [reply index.html]: will do!

**Pull request merged**